sklearn/model_selection/_validation.py

```diff
@@ -106,19 +114,49 @@ def cross_val_score(estimator, X, y=None, groups=None, scoring=None, cv=None,
 
     Returns
     -------
-    scores : array of float, shape=(len(list(cv)),)
+    scores : array of float or dict of such arrays, array shape=(n_splits,)
```

Comment:
We wanted a `cross_validate` function which will return a dict similar to grid search's cv_results... but that's for another PR i suppose...